### PR TITLE
Adds cylayer and cyzone to docstring of AtomGroup.select_atoms

### LINKED ISSUE #1788: Cylayer and Cyzone aren't documented in select_atoms
The docstring for `AtomGroup.select_atoms` doesn't mention cylayer or cyzone.  They are documented [here](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/doc/sphinx/source/documentation_pages/selections.rst) however

### PR DESCRIPTION
**Pull request by palnabarun**
Fixes #1788 

Changes made in this Pull Request:
 - Adds to docstring of `AtomGroup.select_atoms`

PR Checklist
------------
 - [x] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Comment by palnabarun** [comment]
@richardjgowers  Thank you for pointing that out. Made the appropriate changes.

I had a doubt. What is the difference between `versionadded` and `versionchanged` in the docstrings?

**Comment by orbeckst** [comment]
versionadded just states the release when the functionality appeared.

versionchanged states the release when something important was changed and the text explains the changes briefly. Grep through the code to see how we use it.

Do you have a specific question to their usage in this case?

**Comment by palnabarun** [comment]
I grep'ed through the code and saw the ideal usage situations for both of these.

I feel that in this specific case. Neither of those would apply as this is just a documentation change. In reality, no functionality was added or changed. The functionality was already there just not documented. What do you think?

**Comment by richardjgowers** [comment]
@palnabarun yeah this PR needs neither, it just needs the lines of docs adding

**Comment by palnabarun** [comment]
Thank you @richardjgowers  @orbeckst  for the insights. I made the necessary changes.

File: package/MDAnalysis/core/groups.py [hunk 1]
```diff
@@ -1908,6 +1908,22 @@ def select_atoms(self, sel, *othersel, **selgroups):
...
                 *inner radius* of the selection COG
+            cylayer *innerRadius* *externalRadius* *zMax* *zMin* *selection*
+                selects all atoms within a cylindric layer centered in the
+                center of geometry (COG) of a given selection,
+                e.g. ``cylayer 5 10 10 -8 protein`` selects the center of
+                geometry of protein, and creates a cylindrical layer of inner
+                radius 5, external radius 10 centered on the COG. In z, the
+                cylinder extends from 10 above the COG to 8 below. Positive
+                values for *zMin*, or negative ones for *zMax*, are allowed.
+            cyzone *externalRadius* *zMax* *zMin* *selection*
```
Comment (orbeckst) [inline package/MDAnalysis/core/groups.py]: Could you change the order, please?
- sphzone
- sphlayer
- **cyzone**
- **cylayer**

for consistency with `sp*`?

Comment (palnabarun) [reply package/MDAnalysis/core/groups.py]: Sure.

Thank you for pointing this out.

**Pull request merged**